ivy.el

```diff
@@ -253,6 +253,9 @@ It is a list of (CALLER . HEIGHT).  CALLER is a caller of
 If `(minibuffer-depth)' equals this, `ivy-completing-read' will
 act as if `ivy-completing-read-handlers-alist' is empty.")
 
+(defvar counsel--search-commands nil
+  "List of Counsel search commands.")
```

Comment:
Sorry, I didn't realise I was looking at `ivy.el`; I thought this was in `counsel.el`.

Given that this is being used in `ivy.el`, and given that its intented purpose is as an external API, it has to be given the `ivy-` (not `ivy--`) prefix and a name more related to its purpose, e.g. `ivy-highlight-search-commands` (I'm not very good at naming things, perhaps @abo-abo has a better suggestion).